pages/augment/_augment.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <div>
+        <el-tabs type="border-card" v-model="tabIndex">
+            <el-tab-pane :label="seasonInfo.title" disabled>
+
+            </el-tab-pane>
+            <el-tab-pane label="챔피언 목록">
+                <ul>
+                    <li v-for="(unit, u_idx) in $store.state.unitList" :key="unit.id" class="unit-list">
+                        <div class="unit-box">
+                            <div class="unit-img-box">
+                                <img src="~/assets/images/favicon.png" :src="unit.iconPath" alt="챔피언 이미지">
+                            </div>
+                            <div class="unit_name">{{ unit.krName }}</div>
+                        </div>
+                    </li>
+                </ul>
+            </el-tab-pane>
+            <el-tab-pane label="아이템 목록">
+
+            </el-tab-pane>
+            <el-tab-pane label="추천 덱">
+
+            </el-tab-pane>
+            <el-tab-pane label="증강">
+
+            </el-tab-pane>
+            <el-tab-pane label="특성">
+
+            </el-tab-pane>
+
+        </el-tabs>
+
+
+    </div>
+</template>
+
+<script>
+export default {
+    name: "Augment",
+    data() {
+        return {
+            tabIndex: '4',
```

Comment:
Codacy has a fix for the issue: [Replace `'4'` with `"4"`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9690624)
```suggestion
            tabIndex: "4",
```